lib/routes/matters/author.js

```diff
@@ -40,6 +42,7 @@ module.exports = async (ctx) => {
             title: article.title,
             author: user.displayName,
             description: article.content,
+            pubDate: parseDate('article.createdAt'),
```

Comment:
the date is always in ISO format, why nott justt call new Date?

```js
> new Date("2021-07-24T06:30:31.456Z")
2021-07-24T06:30:31.456Z
```

Reply:
It's standard